SECURITY.md

```diff
@@ -0,0 +1,91 @@
+# Security Policy
+
+We appreciate the efforts of security researchers and the broader community in helping us maintain a
+secure environment for our users. If you discover a security vulnerability, we kindly request that
+you report it to us privately before disclosing it publicly. This allows us to address the issue
+promptly and protect our users.
+
+## Reporting Security Issues
+
+**Please do not report security vulnerabilities through public GitHub issues.**
+
+Instead, please report the vulnerability via [https://github.com/LibertyDSNP/frequency/security](https://github.com/LibertyDSNP/frequency/security).
```

Comment:
It's unclear from looking at this page how I would do this... Do we mean to send people to https://github.com/LibertyDSNP/frequency/security/advisories/new?

Reply:
@wesbiggs It looks different for you as you are logged into an account with greater access than others.

This is what it looks like for someone who doesn't have access:
<img width="1739" alt="image" src="https://github.com/LibertyDSNP/frequency/assets/1252199/ec651d85-ea2a-4bec-bde7-7d6f1ce0ac55">

Reply:
However perhaps the `/new` is a better spot considering you are already reading this file?

Reply:
I think there should be clear instructions on how to send an encrypted message in the security policy.  I found it difficult to get the right information to do so and that will deter people from reporting issues.  I am suggesting adding a short paragraph detailing the steps involved.

Reply:
Also, do we have another secure channel like Element?  If not, why not?

Reply:
I think it is reasonable that those who find a security issue can also figure out how to encrypt their data if they need to use email. Writing instructions for the many different systems is not really an option. If they don't want to do that, they can just use the GitHub reporting system.

We don't want to support any other channels at this time as monitoring them is additional load. Email and GitHub cover the most common ways to report issues.

Reply:
> However perhaps the `/new` is a better spot considering you are already reading this file?

I like being as explicit as possible so +1 to this.

Reply:
Applied as requested!